hera_qm/uvflag.py

```diff
@@ -468,6 +465,29 @@ def to_waterfall(self, method='quadmean', keep_pol=True):
         self.history += 'Collapsed to type "waterfall" with ' + hera_qm_version_str
         self.clear_unused_attributes()
 
+    def collapse_pol(self, method='quadmean'):
+        """
+        Collapse the polarization axis using a given method.
+        Args:
+            method: How to collapse the dimension(s)
+        """
+        method = method.lower()
+        avg_f = qm_utils.averaging_dict[method]
+        if self.mode == 'flag':
+            darr = self.flag_array
+        else:
+            darr = self.metric_array
+        if len(self.polarization_array) > 1:
```

Comment:
I understand there is nothing to collapse if `len(self.polarization_array)==1` but is a silent failure like this desirable? Or should there be some kind of warning or error raised? I think an error sounds too serious, but maybe it can raise a warning saying that there is nothing to collapse? Just to let the user know there is nothing to do or that it's not a reasonable call to make?